build.sh

```diff
@@ -111,6 +111,7 @@ if (( ${NUMARGS} == 0 )) || hasArg cuxfilter; then
     echo "8"
     if [[ ${INSTALL_TARGET} != "" ]]; then
         python setup.py build_ext --inplace
+        RAPIDS_DISABLE_CUDA=true \
         python setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
I think we usually indent the line following an explicit break like this.
```suggestion
        RAPIDS_DISABLE_CUDA=true \
            python setup.py install --single-version-externally-managed --record=record.txt
```

Reply:
ok sure